stretch-assignment/digital_timer/index.js

```diff
@@ -0,0 +1,58 @@
+function timer() {
+    let secondOnes = document.querySelector('#secondOnes');
+    let secondTens = document.querySelector('#secondTens');
+    let msHundreds = document.querySelector('#msHundreds');
+    let msTens = document.querySelector('#msTens');
+    
+    var interval = setInterval(function(){
+        switch(msTens.textContent === '-'){
+            case true:
+                return msTens.textContent = 1;
+                break;
+            case false:
+                switch(msTens.textContent === '9'){
+                    case true:
+                        msTens.textContent = 0;
+                        switch(msHundreds.textContent === '-'){
+                            case true:
+                                return msHundreds.textContent = 1;
+                            case false:
+                                switch(msHundreds.textContent === '9'){
+                                    case true: 
+                                    msHundreds.textContent = 0;
+                                    switch(secondOnes.textContent === '-'){
+                                        case true: 
+                                        return secondOnes.textContent = 1;
+                                        case false:
+                                            switch(secondOnes.textContent === '9'){
+                                                case true: 
+                                                secondOnes.textContent = 0;
+                                                secondTens.textContent = 1;
+                                                let redStyle = document.querySelector('.digits');
+                                                redStyle.classList.add('redDigit');
+                                                clearInterval(interval);
+                                                return;
+                                                case false:
+                                                    return ++secondOnes.textContent;
+                                            }
+                                    }
+                                    case false:
+                                        return ++msHundreds.textContent;
+                                }
+                        }
+                        case false:
+                        return ++msTens.textContent;
+                        break;
+
```

Comment:
I would say that this isn't the best way to use a switch statement.  switches are good when you have to value a bunch of different options.  if you have true or false only, then an if statement can do the job. 

Good job getting the timer to work.  There is an issue though with the logic.  when the timer hits 10 seconds, it stops, but when you start it again, it counts from 10 - 20 seconds rather than resetting to 0.  also, at 20 seconds, when you hit start again, it goes back to 10 seconds instead of 0.  also, when you keep on pressing consequtively the start button, the timer no longer is functional, and it isn't incrementing by the second, it goes super fast.

I would reevaluate the logic that you used in your timer.